### PR TITLE
Handle constants deprecated in Python 3.8

### PR DESCRIPTION
**Pull request by nforro**
Fixes https://github.com/packit/specfile/issues/419.

### CODE REVIEW

File: specfile/formatter.py [hunk 1]
```diff
@@ -29,12 +30,14 @@ def format_expression(expression: str, line_length_threshold: int = 80) -> str:
...
-        if isinstance(node, (ast.Constant, ast.NameConstant)):
+        if sys.version_info < (3, 8) and isinstance(node, ast.NameConstant):
             result += repr(node.value)
-        elif isinstance(node, ast.Str):
+        elif sys.version_info < (3, 8) and isinstance(node, ast.Str):
             result += repr(node.s)
-        elif isinstance(node, ast.Num):
+        elif sys.version_info < (3, 8) and isinstance(node, ast.Num):
             result += repr(node.n)
+        elif isinstance(node, ast.Constant):
```
[the remark this answers is not in the document]
Comment (nforro) [reply specfile/formatter.py:39]: That's weird, it was part of the code that run on 3.6 before, unless it was broken the entire time.

[the remark this answers is not in the document]
Comment (nforro) [reply specfile/formatter.py:39]:
```python
$ python3
Python 3.6.8 (default, Sep  9 2024, 06:12:39) 
[GCC 8.5.0 20210514 (Red Hat 8.5.0-22)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import ast
>>> ast.Constant
<class '_ast.Constant'>
```

[the remark this answers is not in the document]
Comment (mfocko) [reply specfile/formatter.py:39]: oh, it is there :eyes: I have a feeling I don't want to know what it means…

**Review by mfocko — APPROVED** [review]

**Comment by hroncok** [comment]
Thank you!

**Pull request merged**